packages/legend-query-builder/src/components/result/QueryBuilderResultPanel.tsx

```diff
@@ -389,7 +391,9 @@ export const QueryBuilderResultPanel = observer(
         : undefined;
       if (_executionResult instanceof TDSExecutionResult) {
         const rowLength = _executionResult.result.rows.length;
-        return `${rowLength} row(s)${
+        const truncatedRowLength =
```

Comment:
this shouldnt be needed anymore right? since we remove the extra value in `runQuery` ?